admin_manual/issues/code_signing.rst

```diff
@@ -47,7 +47,7 @@ Is Code Signing Mandatory For Apps?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Code signing is optional for all third-party applications. Applications
-with a tag of "Official" on apps.owncloud.com require code signing.
+with a tag of "Official" on apps.nextcloud.com require code signing.
```

Comment:
With Nextcloud App Store - I think **all** apps (not only official ones) require code-signing. @BernhardPosselt